src/view/com/composer/Composer.tsx

```diff
@@ -744,14 +750,24 @@ let ComposerPost = React.memo(function ComposerPost({
 
   const onPhotoPasted = useCallback(
     async (uri: string) => {
-      if (uri.startsWith('data:video/')) {
-        onSelectVideo(post.id, {uri, type: 'video', height: 0, width: 0})
+      if (uri.startsWith('data:video/') || uri.startsWith('data:image/gif')) {
+        if (isNative) return // web only
```

Comment:
is it possible to paste videos on native? do we need to handle this? it will be broken in prod anyway since the old code sets the height/width to 0